api/inventorypb/services.proto

```diff
@@ -468,6 +468,20 @@ message RemoveCustomLabelsRequest {
 
 message RemoveCustomLabelsResponse {}
 
+message UpdateServiceRequest {
```

Comment:
Since we have `ChangeXXX` naming for agents, I would suggest having the same naming for services and nodes to keep it consistent

Reply:
And TBH we might need to separate these endpoints for each service type since other features Inventory Management is coming.

Reply:
Splitting as in MySQL, MongoDB, Postgres, etc?

Reply:
I suppose so but doesn't have a strong opinion about that. Not sure if we will require it at all.

Reply:
I suggest we stick with `ChangeService` then.
These are common settings for any service. Similar to custom labels.

Reply:
ok